RP-1.netkan

```diff
@@ -2,7 +2,7 @@
 	"spec_version" : "v1.26",
 	"identifier" : "RP-1",
 	"name"       : "Realistic Progression One (RP-1)",
-	"abstract"   : "Realistic Progression One - Career for Realism Overhaul",
+	"abstract"   : "Realistic Progression One - Career for Realism Overhaul. Do not install this without the Express Install unless you know what you're doing.",
```

Comment:
I'd probably say something more like "Don't install this directly, instead install the RP-1 Express Install except in rare circumstances."

Reply:
And I'd also remove the repeat of RP-1, so it's just "Career for Realism Overhaul. Don't..." because you _do_ indeed want the "Don't" bit to show up on the first line.

Reply:
> I'd probably say something more like "Don't install this directly, instead install the RP-1 Express Install except in rare circumstances."

This is actually a fair bit longer, and I'd argue that "unless you know what you're doing" is a better descriptor than "rare circumstances". I'll put directly in there though.

> And I'd also remove the repeat of RP-1, so it's just "Career for Realism Overhaul. Don't..." because you do indeed want the "Don't" bit to show up on the first line.

Good point, I'll go ahead and do that.